Library/Formula/node.rb

```diff
@@ -55,42 +56,50 @@ def install
     system "./configure", *args
     system "make", "install"
 
-    resource("npm").stage libexec/"npm" if build.with? "npm"
-  end
+    if build.with? "npm"
+      resource("npm").stage libexec/"npm_install"
 
-  def post_install
-    return if build.without? "npm"
+      # make sure npm can find node
+      ENV["PATH"] = "#{Formula["node"].prefix}/bin:#{ENV["PATH"]}"
```

Comment:
Mike previously [committed](https://github.com/Homebrew/homebrew/commit/cd21a535b2939b5c02abe60510fbcb4ccc700306) this particular line's wording; I wasn't sure if the not prepending with the normal syntax was a deliberate thing or not.